data/mods/Arcana/tool_armor.json

```diff
@@ -266,23 +254,15 @@
         "use_action" : {
             "type" : "consume_drug",
             "activation_message" : "A strange energy radiates from the ring's gem, spreading a calming sensation over you.",
-            "effects" : [
-                {
-                   "id": "cleric_warding",
-                    "duration": 250
-                }
-            ]
+            "effects" : [ { "id" : "cleric_warding", "duration" : 250 } ]
         },
         "artifact_data" : {
           "effects_worn": ["AEP_PSYSHIELD", "AEP_STR_DOWN"]
         },
         "color" : "white",
-        "encumbrance" : 0,
         "price" : 30000,
         "material" : ["silver", "essencemat"],
-        "coverage" : 0,
```

Comment:
I see no default value for ```coverage``` defined, so removing of this line causes an error. Please run Cataclysm as  ```./cataclysm --check-mods``` (if you are on Linux) for testing your changes.